tools/lib/logreader.py

```diff
@@ -44,17 +44,13 @@ def __init__(self, fn, canonicalize=True, only_union_types=False, sort_by_time=F
     if ext == ".bz2" or dat.startswith(b'BZh9'):
       dat = bz2.decompress(dat)
 
-    ents = capnp_log.Event.read_multiple_bytes(dat)
-
-    _ents = []
     try:
-      for e in ents:
-        _ents.append(e)
```

Comment:
did the previous method read messages up until it got a corrupted one? that can be useful

Reply:
Yes, the previous code is more robust